### DISCUSSION TITLE
Argo Rollout installation config for Prod clusters

### DISCUSSION THREAD

**Discussion by Sonam11** [argoproj/argo-rollouts#1257] [Q&A]
Hi,
We are exploring Argo Rollouts to use for deployment. I have a query that to use Argo Rollouts in production K8S controller and suppose we have 100 deploy requests per day, then how much should be CPU and memory requirements (for an average case) for Argo Rollouts. 

Please help.

**Comment by kostis-codefresh** [comment]
Without having any hard numbers at hand, I would say that in most cases the amount of resources the controller needs is much less than the resources needed by the application deployed.

It is more important to account for the application resources. For example, if you use blue/green deployments, you will get 2x the number of pods while the deployment is running. Canaries will also increase the number of pods (old version + new version). If you accommodate these resources, I don't think the controller itself will pose an issue for you.
